### PR TITLE
Add DG: planned enhancement for eremovev

### PR DESCRIPTION
**Pull request by ruo-x**
covers this point:
when u remove the volunteer from an event, the volunteer will still be in the list, only after u call elistv again, the volunteer will then be removed

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1763,3 +1763,9 @@ The current implementation of the name format feature only supports alphanumeric
 
 ### Change `eaddm` command format to maintain consistency
 The current implementation of `eaddm` is as follows: `eaddm eid/EVENT_INDEX m/MATERIALS_AND_LOGISTICS_NEEDED`. However, this is inconsistent with other Event commands that only involve a single event, like `edelete` or `eshow`, where the index is directly stated instead of using the `eid/` parameter. This will be fixed eventually to improve usability for the app and prevent inconsistency in command formats for features.
+
+### Improve list panel display for Event Remove Volunteer Feature
+If the volunteer list panel is currently displaying the volunteers assigned to a particular event (used `elistv` command), performing an `eremovev` command to remove a volunteer from the event should result in the removal of this volunteer from the displayed list. <br>
+However, our implementation only updates this displayed list if the user executes the `elistv` command for the same event again. 
+The same occurrence also happens when the event list panel is currently displaying the events joined by a particular volunteer (used `vliste` command), and an `eremovev` command is executed to remove this volunteer from the event.
```
Comment (tiongMax) [inline docs/DeveloperGuide.md:1770]: u tried it out ady ah?

**Comment by tiongMax** [comment]
Add new lines for event and volunteer storage data file and dg to resolve ci issues

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jordankanghm) [reply docs/DeveloperGuide.md:1770]: eh u guailan ah didnt u find this bug

**Review by jordankanghm — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md:1770]: ya tried it out le

Comment (tiongMax) [reply docs/DeveloperGuide.md:1770]: u need more glasses or expand ur brain

> eh u guailan ah didnt u find this bug

**Review by tiongMax — APPROVED** [review]
lgtm!

**Pull request merged**